python/shotgun_model/shotgun_model.py

```diff
@@ -440,6 +440,7 @@ def _load_data(
                                           each column in the row contains the value for the corresponding field from columns. This means
                                           that the data from the loaded entity will be available field by field. Subclasses can modify
                                           this behavior by overriding _get_additional_columns.
+        :param editable_columns:          A subset of ``columns`` that will be editable in views that use this model.
```

Comment:
I've added an example and the test app ticket now suggests we link to that code once we have examples in place.